solutions/src/main/java/ru/sberbank/school/task02/ConsoleClient.java

```diff
@@ -0,0 +1,129 @@
+package ru.sberbank.school.task02;
+
+import org.apache.commons.cli.*;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.FxRequest;
+import ru.sberbank.school.task02.util.FxResponse;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConsoleClient implements FxClientController {
+    private CommandLineParser cmdParser;
+    private Options cmdOptions;
+    private FxConversionService converter;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss");
+
+    @Override
+    public List<FxResponse> fetchResult(List<FxRequest> requests) {
+        if (requests.size() <= 0) {
+            return null;
+        }
+        List<FxResponse> result = new ArrayList<>();
+        for (FxRequest request : requests) {
+            result.add(fetchResult(request));
+        }
+        return result;
+    }
+
+    public ConsoleClient(ExternalQuotesService service) {
+        converter = new Converter(service);
+        cmdParser = new DefaultParser();
+        cmdOptions = new Options();
+        cmdOptions.addOption("o", "operation", true, "ClientOperation");
+        cmdOptions.addOption("s", "symbol", true, "Symbol");
+        cmdOptions.addOption("a", "amount", true, "Amount");
+    }
+
+    @Override
+    public FxResponse fetchResult(FxRequest request) {
+        Symbol symbol = getSymbol(request.getSymbol());
+        ClientOperation operation = getOperation(request.getDirection());
+        BigDecimal amount = getAmount(request.getAmount());
+        if (symbol == null || amount == null || operation == null || amount.compareTo(BigDecimal.ZERO) <= 0) {
+           return makeResponse(request,null);
```

Comment:
&#39;if&#39; child has incorrect indentation level 11, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a322397813>*